qdarkstyle/utils/__main__.py

```diff
@@ -50,28 +52,120 @@ def __init__(self, parser_args):
 
     def on_modified(self, event):
         """Handle file system events."""
-        if event.src_path.endswith('.qss'):
+        if event.src_path.endswith("palette.py"):
             # TODO: needs implementation for new palettes
-            process_palette(compile_for=self.args.create)
-            print('\n')
+            for palette in [DarkPalette, LightPalette]:
+                process_palette(palette=palette, compile_for=self.args.create)
+            _logger.info("\n")
+
+
+# Based on https://sumit-ghosh.com/posts/parsing-dictionary-key-value-pairs-kwargs-argparse-python/
+class CustomPaletteParser(argparse.Action):
```

Comment:
```suggestion
class CustomPaletteParser(argparse.Action):
    """
    This is based on the solution found at https://sumit-ghosh.com/
    posts/parsing-dictionary-key-value-pairs-kwargs-argparse-python
    """

```

- Move comment to docstring.
- Make docstring fit in 79 columns.

Reply:
Following the removal of `--custom-palette-key-value` this should be removed, no?

Reply:
Not totally sure if you read this comment @ccordoba12 Let me know if this class should be removed